docs/layouts/shortcodes/region_ips.html

```diff
@@ -0,0 +1,81 @@
+<!-- INDIVIDUAL REGION TABLE SHORTCODE -->
```

Comment:
I find Markdown to be much more readable than HTML, usually even with tables.

Reply:
I get you - some of the feedback in these sections we've gotten from other teams here is that MD tables get messy, and difficult to format. Part of the reason I split off into a data file + shortcode so they didn't have to deal with it. 

Aside from that, I was trying to clean up while preserving individual region anchors, in case someone was used to linking a specific region (AU-2, for example).

Reply:
I definitely like the use of YAML (although with all the quotes it looks more like JSON :see_no_evil: ). I was just thinking even the template could be Markdown, but it's not a big deal.